### PR TITLE
Feat3 video listing

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review. Added
1. VideoCard
2. Fetched data from backend
3. Explore Page
image

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

**Review by Varu98 — APPROVED** [review]
Looks Good to merge 👍🏻

File: src/Components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { DropdownPanel } from "../components";
+import "./VideoCard.css";
+
+function VideoCard({ video }) {
+  console.log("And here?");
+  console.log(video);
```
Comment (ashishkatiyar2404) [inline src/Components/VideoCard/VideoCard.jsx:7]: you can remove console

**Review by ashishkatiyar2404 — APPROVED** [review]
code looks great.
suggestions: - you can remove comments

File: src/Components/VideoCard/VideoCard.jsx (hunk 1 above)
Comment (ShrutiLakhani) [reply src/Components/VideoCard/VideoCard.jsx:7]: Thant You. Will add Toast with other features

**Comment by ShrutiLakhani** [comment]
Thank you all for review. Merging to Dev

**Pull request merged**